### PR TITLE
Create cancel button

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?

It creates a cancel button.

#### Ticket

https://trello.com/c/Wh5EvCc1
#### Screenshots
captured (1)

### CODE REVIEW

**Review by MutegekiHenry — APPROVED** [review]
LGMT

**Comment by colinwagaba** [comment]
can't seem to test this locally, even after pulling the branch, any help?

**Comment by neelxie** [comment]
> can't seem to test this locally, even after pulling the branch, any help? @neelxie my guess is i import it into any component?

@colinwagaba of course, since it is a standalone component and not a page, you would probably add it into the information bar for the projects page like I did to test it.

**Comment by colinwagaba** [comment]
nice

File: src/components/CancelButton/CancelButton.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.CancelButton {
+  width: 56px;
```
Comment (colinwagaba) [inline src/components/CancelButton/CancelButton.css:2]: we still need to work on the units... from px to em. my guess is we shall do this in the refactor

**Review by colinwagaba — APPROVED** [review]

**Pull request merged**